docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,20 @@
+# Security and Permissions
+
+## How we keep Renovate safe to use
+
+...
+
+## Permission comparison
+
+| Permission        | Renovate hosted app |  Forking Renovate  | Why                                                 |
+| ----------------- | :-----------------: | :----------------: | --------------------------------------------------- |
+| Dependabot alerts |       `read`        |       `read`       | Create security updates                             |
+| Administration    |       `read`        |       `read`       | ...                                                 |
+| Metadata          |       `read`        |       `read`       | Get basic repository information                    |
+| Checks            | `read` and `write`  |   not applicable   | ...                                                 |
+| Code              | `read` and `write`  |       `read`       | Find Renovate config file and package manager files |
+| Commit statuses   | `read` and `write`  | `read` and `write` | ...                                                 |
+| Issues            | `read` and `write`  | `read` and `write` | Create dependency dashboard                         |
+| Pull Requests     | `read` and `write`  | `read` and `write` | Create update PRs                                   |
+| Workflows         | `read` and `write`  |   not applicable   | Update dependencies in workflow files               |
+| Deployments       |   not applicable    |       `read`       | ...                                                 |
```

Comment:
I think this is a mistake actually, can be removed

Reply:
> I think this is a mistake actually, can be removed

Are you sure about removing it from the table? Here's a screenshot from the current Forking Renovate app installation screen:

![forking-renovate-install-screen](https://user-images.githubusercontent.com/34918129/152768832-a15c062c-6a75-41a9-a7c7-ed253d9dbbc3.png)

Reply:
I added alerts and removed deployments